build.rs

```diff
@@ -1,3 +1,10 @@
+// Copyright (c) 2022 Jan Holthuis <jan.holthuis@rub.de>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy
+// of the MPL was not distributed with this file, You can obtain one at
+// http://mozilla.org/MPL/2.0/.
+//
+// SPDX-License-Identifier: MPL-2.0
 use glob::glob;
```

Comment:
I think it would be a nice addition to `sourceheaders` if it put at least one newline after the license header.

Reply:
Or you can just do that yourself I guess since its purely a style choice.

Reply:
Done.